tests/code/test_program_eligibility.py

```diff
@@ -0,0 +1,42 @@
+import pytest
+from app.models.eventParticipant import EventParticipant
+from app.controllers.events.volunteerRegisterEvent import volunteerRegister
+from peewee import DoesNotExist
+from app.models.user import User
+from app.models.event import Event
+from app.controllers.events.meetsReqsForEvent import isEligibleForProgram
+
+@pytest.mark.integration
+def test_noUserVolunteerRegister():
```

Comment:
It's good that you are testing all of these cases, however a function called `isEligibleX` should really be returning True or False, and not using exceptions to handle normal False cases.

Reply:
This function was to test if the user does not exist and if the program does not exist.